aldryn_faq/tests/test_base.py

```diff
@@ -42,7 +48,28 @@ def assertCountEqual(a, b):
         return assertCountEqual(a, b)
 
 
-class AldrynFaqTestMixin(TestUtilityMixin, object):
+class CleanUpMixin(object):
+
+    def reload_urls(self):
+        """
+        Clean up url related things (caches, app resolvers, modules).
+        Taken from cms.
+        :return: None
+        """
+        clear_app_resolvers()
+        clear_url_caches()
+        url_modules = [
+            'cms.urls',
+            'aldryn_events.urls',
```

Comment:
this should be `aldryn_faq` instead of `aldryn_events`

Reply:
changed to `aldryn_faq`